website/blog/2023-08-14-aa-compliance-accessibility.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "AA Compliant Website Accessibility through Wavemaker Platform"
+author: Bhargavi Gunda
+---
+
+On a low-code platform, achieving AA compliance requires a concerted effort to implement various accessibility standards and best practices. By following WCAG guidelines, using semantic markup, prioritizing keyboard accessibility, ensuring color contrast, providing alternative text for images, and implementing ARIA attributes, developers can create an inclusive and accessible platform for users of all abilities. Continuous testing, documentation, and training further contribute to the platform's commitment to accessibility and enable a seamless and enriching experience for all users.
```

Comment:
We should make clear this is for apps developed with WaveMaker and not the studio itself. So instead of starting with "On a low-code platform" say "Enabling AA Compliance for web apps built with WaveMaker.."

Reply:
This is done